starknet_devnet/starknet_wrapper.py

```diff
@@ -247,8 +247,12 @@ async def store_deploy_transaction(self, contract_address: str, calldata: List[i
             contract_address_salt=hex(salt)
         )
 
-    async def store_invoke_transaction(self, contract_address: str, calldata: List[int], entry_point_selector: int, status: TxStatus, error_message: str=None):
-        return await self.store_transaction(
+    # pylint: disable=too-many-arguments
```

Comment:
Maybe the argument could be a single Object/Dataclass with all these arguments as properties